src/rospkg/environment.py

```diff
@@ -109,7 +109,7 @@ def get_ros_package_path(env=None):
     """
     if env is None:
         env = os.environ
-    return env.get(ROS_PACKAGE_PATH, None)
+    return env.get(ROS_PACKAGE_PATH, '') + ':/usr/share'
```

Comment:
* William Woodall <notifications@github.com> [2016-08-10 15:53]:
>   - `ROS_NOT_DEBIAN` is set in the environment (this could be set in `/opt/ros/kinetic/setup.*sh`)

I would propose to name it `RPP_NO_DEFAULTS`, as this is not about
Debian ;). Also I see this similar to `gcc -nostdinc` and alike. If you
agree, I will implement this tomorrow.